etc/kayobe/inventory/group_vars/monitoring/firewall

```diff
@@ -0,0 +1,83 @@
+---
+
+firewallgen_enable_firewall: True
+
+firewallgen_ipv4_input_allow_rules:
+  - interface: "lo"
+    port: 25
+    proto: tcp
+    destination: "127.0.0.1"
+    comment: "hint: used by 'master'"
+  - interface: "{{ admin_oc_net_interface }}"
+    port: 22
+    proto: tcp
+    comment: "hint: used by 'sshd'"
+  - interface: "{{ internal_net_interface }}"
+    port: 3000
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'grafana-server' in docker container 'grafana'"
+  - interface: "{{ internal_net_interface }}"
+    port: 5601
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'node' in docker container 'kibana'"
+  - interface: "{{ internal_net_interface }}"
+    port: 9091
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'prometheus' in docker container 'prometheus_server'"
+  - interface: "{{ internal_net_interface }}"
+    port: 9093
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'alertmanager' in docker container 'prometheus_alertmanager'"
+  - interface: "{{ internal_net_interface }}"
+    port: 9100
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'node_exporter' in docker container 'prometheus_node_exporter'"
+  - interface: "{{ internal_net_interface }}"
+    port: 9198
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'openstack-expor' in docker container 'prometheus_openstack_exporter'"
+  - interface: "all"
+    port: 123
+    proto: udp
+    comment: "hint: used by 'ntpd'"
+  - interface: "breno3.206"
```

Comment:
looks like storage_mgt_net, but I suspect we only what ntp down the admin_oc again.

Curious we don't see this on the controllers.